### DISCUSSION TITLE
Sanity check me on the multiroom_player entity

### DISCUSSION THREAD

**Discussion by secbitchris** [kalkih/mini-media-player#645] [Q&A]
I have HA, Sonos, HACS, have a seemingly functional installation of mini-media-player. My goal is to put 3 sonos speakers into a group so I can control their audio as one. I eventually want to use this group in an automation on my HASPone device so I can use one volume slider to control all 3.

What am I doing wrong?

Here is my code.

I am presuming the entity of media_player.multiroom_player is some sort of built-in way of putting stuff into a group? Although it doesn't auto-populate when I manually type it. Is this what I have to have to make that happen or is there some other thing it has to be called?

- type: entities
  entities:
    - type: custom:mini-media-player
      entity: media_player.multiroom_player
      group: true
      source: icon
      info: short
      hide:
        volume: true
        power: true
    - type: custom:mini-media-player
      entity: media_player.studio_1
      group: true
      hide:
        controls: true
    - type: custom:mini-media-player
      entity: media_player.studio5
      group: true
      hide:
        controls: true
    - type: custom:mini-media-player
      entity: media_player.studioplay3
      group: true
      hide:
        controls: true

I get this:

Configuration errors detected:
No type provided.

**Comment by kalkih** [comment]
The initial dash (`-`) in `- type: entities` and you should be good.

The `media_player.multiroom_player` is not some built in entity, in the example from the readme it was a multiroom amplifier, with the outputs configured as media players.

Depending on what you want to achieve, one way would be:
Speaker group management setup e.g. with Sonos, check out [this example](https://github.com/kalkih/mini-media-player#speaker-group-management). You could then ise the option `sync_volume` in the [speaker group configuration](https://github.com/kalkih/mini-media-player#speaker-group-object) in order to adjust the volume level of all grouped speakers with the volume slider.

**Reply by ZNicoZ** [reply]
With `sync_volume` all speakers are changed to the same volume when before one speaker was on 20% and another on 40%, the relation of volume between the speakers is lost. :(

**Comment by secbitchris** [comment]
Thanks! I also see helpers may give a way to do this too. Problem is, the latest update broke spotify and sonos ENTIRELY, so I am going to have to wait some weeks or months until HA fixes those.
